policykit/policykit/settings.py

```diff
@@ -239,6 +239,7 @@
 
 from celery.schedules import crontab
 
+# Replace with "amqp://USERNAME:PASSWORD@localhost:5672/VIRTUALHOST"
```

Comment:
This belongs with  https://github.com/amyxzhang/policykit/pull/364 - now that both Metagov and PolicyKit are using celery, we need to use a RabbitMQ virtual host as the broker